.github/workflows/container.yml

```diff
@@ -0,0 +1,58 @@
+name: Container
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - '@lagon/serverless@*'
+  pull_request:
+
+env:
+  REGISTRY: ghcr.io
```

Comment:
Could we publish to Docker Hub instead? I guess we'll need a Docker token

Reply:
sure. I just used githubs registry as I wasn't needing to create a token for it and I somehow trust it a bit more atm with some of the things going on at docker hub 🙈 (personal opinion)

Reply:
@QuiiBz Could you  create a token for Docker Hub and save it as secret as `DOCKER_HUB_TOKEN` and tell me the username / org name you want to use.

Reply:
Done for the secret, and the [image is here](https://hub.docker.com/r/lagonapp/lagon):
- User is `lagonapp`
- Image name is `lagon`

Reply:
@QuiiBz Thanks. As there probably needs to be at least a second image for the dashboard for having a complete `docker-compose` stack for self-hosting it might be better to call the image `serverless` or `lagon-serverless`?

Reply:
Hum agree, we can go with `lagonapp/serverless` and `lagonapp/dashboard`

Reply:
okay, then the code should be ready from my side.